ginga/gw/Desktop.py

```diff
@@ -227,7 +228,7 @@ def create_toplevel_ws(self, wsname, width, height,
 
         if x is not None:
             root.move(x, y)
-        return bnch
+        #return bnch  # TODO: Undefined bnch!
```

Comment:
`bnch` is undefined in this method, and I can't figure out where it is supposed to come from, so I commented this.